articles.tex

```diff
@@ -97,11 +97,10 @@
 A quorum of fifty percent of the Total Number of Possible Votes is required for passage.
 
 \asubsection{Semantic Changes}
-Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
-Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted in the House during the week.
-The final proposal is presented the following week at the end of House Meeting.
-Attendance for the discussion on the change is not required, but only members who stay until the end of discussion will receive ballots.
-At the end of discussion, ballots are distributed for a ballot House vote as described in \ref{Balloted Vote}.
+Any semantic change to the Constitution requires the change to be brought up at House Meeting and proposed in writing for discussion at the following House Meeting.
+The text of the change may be modified up until the aforementioned House Meeting. 
+At the beginning of the discussion period, ballots are distributed for a balloted House vote as described in \ref{Balloted Vote}.
+Attendance for the discussion on the change is not mandatory.
```

Comment:
This is still really vague. No clue what this does for quorum. If attendance is not mandatory, do I still get a ballot? What happens if down the line someone reads this and thinks it means that "attendance for the discussion on the change is not mandatory" means "it is up to the chair to host discussion as it is optional." 

Might want to mention that discussion for changes on amendments happen at the end of house meeting. Not really necessary though. 

Also, probably want to refer to it as the "proposed amendment" instead of "the change"